model/Licensing/Properties/isDeprecatedLicenseId.md

```diff
@@ -4,11 +4,26 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+isDeprecatedLicenseId specifies whether a license or exception identifier
+has been marked as deprecated.
 
 ## Description
 
-TODO
+The isDeprecatedLicenseId property specified whether an identifier for a
+License or LicenseException has been marked as deprecated. If the property
+is not defined, then it is presumed to be false (e.g., not deprecated).
+
+If the License or LicenseException is included on the SPDX License List, then
+the `deprecatedVersion` property should indicate which version release of the
```

Comment:
Fair question! I believe we have a `deprecatedVersion` for all deprecated licenses and exceptions, so this can be "indicates" rather than "should indicate". I'll update accordingly.